### PR TITLE
Edited the Sentinel-2 True Colour custom script to also include current EOB Default

### PR DESCRIPTION
**Pull request by azlinszkysinergise**
The current default True Color evalscript for EO Browser was not included on the Custom Scripts Repo, and is different from the True Color used in Copernicus Browser.
I have added the EOB default script and some images and text explaining the differences.

### CODE REVIEW

File: sentinel-2/true_color/README.md [hunk 1]
```diff
@@ -5,6 +5,11 @@ grand_parent: Sentinel
 layout: script
 permalink: /sentinel-2/true_color/
 nav_exclude: true
+scripts:
+- - Visualization
+  - script.js
+- - EO Browser
+  - eob.js
 examples:
 - zoom: '10'
```
[the remark this answers is not in the document]
Comment (azlinszkysinergise) [reply sentinel-2/true_color/README.md:14]: done

File: sentinel-2/true_color/eob.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+function setup() {
+   return {
+    input: ["B04", "B03", "B02","dataMask"],
+    output: { bands: 4 }
+  };
+}
+
+function evaluatePixel(samples) {
+    let val = [samples.B04, samples.B03, samples.B02,samples.dataMask];
```
[the remark this answers is not in the document]
Comment (azlinszkysinergise) [reply sentinel-2/true_color/eob.js]: done

File: sentinel-2/true_color/eob.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+let viz = new HighlightCompressVisualizer(minVal, maxVal);
+
+function setup() {
+   return {
+    input: ["B04", "B03", "B02","dataMask"],
+    output: { bands: 4 }
+  };
+}
+
+function evaluatePixel(samples) {
```
[the remark this answers is not in the document]
Comment (azlinszkysinergise) [reply sentinel-2/true_color/eob.js:14]: I am keeping the exact code used for EO Browser as much as possible, therefore I leave it as "samples".

File: sentinel-2/true_color/README.md [hunk 4]
```diff
@@ -41,6 +70,21 @@ True color visualization of Rome, on 8.10.2017.
 
 ![True color visualization of Rome, on 8.10.2017.](fig/fig1.png)
```
[the remark this answers is not in the document]
Comment (azlinszkysinergise) [reply sentinel-2/true_color/README.md:71]: done

File: sentinel-2/true_color/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (chorng) [reply sentinel-2/true_color/README.md:14]: There are two sets of links to EOB and Copernicus Browser. I think you need to remove line 14 to line 26.

File: sentinel-2/true_color/README.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (chorng) [reply sentinel-2/true_color/README.md:71]: Line 69 and 71 also need to be deleted.

Screenshot 2024-06-21 at 10 40 27

[the remark this answers is not in the document]
Comment (azlinszkysinergise) [reply sentinel-2/true_color/README.md]: Done

**Comment by azlinszkysinergise** [comment]
I have checked the requested changes (I thought they were already in the previous version, but maybe something went wrong) and pushed again

**Review by chorng — APPROVED** [review]
Everything looks good!

**Pull request merged**